tools/site_admin/copy_pages.php

```diff
@@ -651,13 +654,14 @@ function do_stuff( $projectid_, $from_image_, $page_name_handling,
         foreach ( $subscribable_project_events as $event => $label )
         {
             $query = sprintf("
-                      SELECT username FROM user_project_info
-                      WHERE projectid = '%s' AND
-                            iste_$event = 1",
-                    mysqli_real_escape_string(DPDatabase::get_connection(), $projectid_['from'])
+                SELECT username FROM user_project_info
+                WHERE projectid = '%s' AND
+                    iste_%s = 1",
```

Comment:
Keep `$event` in the string since it is a column name. We aren't escaping it or anything anyway.